index.ts

```diff
@@ -3,186 +3,36 @@
  * SPDX-License-Identifier: MIT
  */
 
-import express, { Request, Response } from 'express';
-import bodyParser from 'body-parser';
+import {NextFunction, Request, RequestHandler, Response} from "express";
 
-/**
- * Endpoint argument definition, defines a argument value to a request, like a path, query, header or body parameter
- * @typedef {Object} RouteEndpointArgument
- * @property {string} name - name of the argument
- * @property {string} transport - path, query, header, body
- * @property {string} [argument] - in case of header transport, this is the header name like X-Kapeta-Tags
- */
-export interface RouteEndpointArgument {
-    name: string;
-    transport: string;
-    argument?: string;
-}
 
-/**
- * Endpoint definition
- * @typedef {Object} RouteEndpoint
- * @property {string} method - GET, POST, PUT, DELETE etc.
- * @property {string} path - /some/path/{id}/{type}
- * @property {string} [description]
- * @property {RouteEndpointArgument[]} arguments
- * @property {function} handler
- */
-export interface RouteEndpoint {
-    method: string;
-    path: string;
-    description?: string;
-    arguments: RouteEndpointArgument[];
-    handler: (...args: any[]) => any;
+declare global {
+    export namespace Express {
+        export interface Response {
+            sendError(err:RESTError|Error|string|any, statusCode?:number):void
```

Comment:
When `any` is part of a union type it overrides the other types in the union, making the union redundant.

Reply:
Yea it's more of the communicative part of it